buildchain/buildchain/docker_command.py

```diff
@@ -237,3 +237,66 @@ def __call__(self) -> Optional[TaskError]:
             command=self.command,
             **run_config
         )
+# The call method is not counted as a public method
+# pylint: disable=too-few-public-methods
```

Comment:
To disable the warning only for the class, the correct syntax is:

```
class DockerTag:
    # pylint: disable=too-few-public-methods
    """A class to expose the `docker tag` command through the API client."""
```

(yeah it was wrong before, we noticed later in another Jordi's PR).